lib/plug/csrf_protection.ex

```diff
@@ -123,6 +140,7 @@ defmodule Plug.CSRFProtection do
   ## Plug
 
   @behaviour Plug
+  @digest "SFMyNTY."
```

Comment:
This could be any string, right? Is there a preference for a shorter or longer string?

Reply:
Nevermind, I see from where it comes from. :) It is HS256 encoded.

Reply:
I also had doubts with this one. I think a comment will be helpful :)